node_frequency: 10
# topics:
#   subscribers:
#     neighbor_target_pdf: "/slap/neighbor/target/pdf"
#     target_range:  "/sensors/usbl_fix"
#     neighbor_pos: "/slap/neighbor/position"
#     vehicle_state: "/nav/filter/state"
#   publishers:
#     target_pdf: "/slap/internal/target/pdf"
#     target_pdf_to_neighbor: "/slap/internal/target/pdf_to_neighbor"
#     estimated_target_to_console: "/Virtual/State"
#     dekf_etc_info: "/slap/dekf/etc_info"
#   services:
#     start_ekf: "/slap/start_ekf"
#     stop_ekf: "/slap/stop_ekf"
#     start_dekf: "/slap/start_dekf"
#     stop_dekf: "/slap/stop_dekf"
#     set_target_pdf: "/slap/set_target_pdf"
#     set_matrices_QR: "/slap/set_matrices_QR"
#     set_ETC_parameter: "/slap/dekf/set_ETC_parameter"
#     set_target_depth: "/slap/dekf/set_target_depth"



Veh_ID: 0
adj_matrix: [0, 1, 
             1, 0]
state_offset: [4290797.543,  491941.239 , 0.0, 0.0]   #old
ini_target_state: [ 10.0,   -5.0,    0.0,    0.0 ]
ini_target_cov:   [ 10.0,   0.0,    0.0,    0.0, 
                    0.0,   10.0,    0.0,    0.0,
                    0.0,   0.0,    0.1,   0.0,
                    0.0,   0.0,    0.0,    0.1 ]
target_depth: 0.0

# threshold function of ETC is: g = c0 + c1e^(-c2*t)
c0: 0
c1: 0
c2: 0
